ouroboros-network-framework/ouroboros-network-framework.cabal

```diff
@@ -72,8 +67,9 @@ library
                      , dns
                      , iproute
 
-                       -- Remove once example codecs are moved into the tests
+                       -- Used by the handshake protocol for now
```

Comment:
If you agree add a comment that we should use `cardano-binary` instead.  This  will be more consistent with other repos (e.g. `cardano-ledger).